### PR TITLE
Pushing the image to GHCR.io

### PR DESCRIPTION
**Pull request by Praveenraj-K**
**Added**

- Pushing the image to GHCR.io

### CODE REVIEW

File: .github/workflows/build-and-publish.yml
Comment (hobbitronics) [inline .github/workflows/build-and-publish.yml:33]: I'm not sure when we use `repository` vs `repository_owner`, is there a hard fast rule?

**Review by hobbitronics — APPROVED** [review]
Looks good. Feel free to wait for another review as I'm not 100% on all details.

File: .github/workflows/build-and-publish.yml
Comment (briskt) [reply .github/workflows/build-and-publish.yml:33]: I think this is correct since we want the tag to be `ghcr.io/silinternational/sync-s3-to-b2`.

File: .github/workflows/build-and-publish.yml
Comment (briskt) [inline .github/workflows/build-and-publish.yml]: Indentation style in this file (and pretty much all of our YAML files) is 2 spaces.
```suggestion
        with:
          registry: ghcr.io
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

**Review by briskt — CHANGES_REQUESTED** [review]

**Review by briskt — APPROVED** [review]

**Pull request merged**
